styles/src/themes.ts

```diff
@@ -188,6 +191,9 @@ export const CONTRAST: Theme = {
   state_label: "#777777",
   state_label_halo: "#ffffff",
   country_label: "#9590aa",
+
+  housenumbers_label: "#91888b",
```

Comment:
I used the same values as `roads_label_minor` for all themes... Looks okay on my taste...
<img width="2560" alt="Screenshot 2024-12-14 at 09 53 14" src="https://github.com/user-attachments/assets/11eb6259-3e58-40f3-a482-6f84f31f6e4b" />